connections/tests/factories.py

```diff
@@ -168,7 +170,9 @@ class Meta:
     project_city = "Helsinki"
     project_district = fuzzy.FuzzyText()
     project_realty_id = fuzzy.FuzzyText()
-    project_new_development_status = fuzzy.FuzzyText()
+    project_new_development_status = fuzzy.FuzzyChoice(
+        ["UNDER_PLANNING", "PRE_MARKETING", "UNDER_CONSTRUCTION", "READY_TO_MOVE"]
```

Comment:
Hmm.. this feels a bit like unnecessary hard-coding. Can we use here ex. `[e.name for e in NewDevelopmentStatusChoices]` or `NEW_DEVELOPMENT_STATUS_MAPPING.keys()`? Same also here https://github.com/City-of-Helsinki/apartment-application-service/pull/34/files#diff-a798cb7a83ba7f20be10e85d8313d1821a2a048fccb045d28d9a7ded59eefacfR50